src/modules/keyboardmanager/KeyboardManagerEditor/KeyboardManagerEditor.exe.manifest

```diff
@@ -7,7 +7,7 @@
 	</application>
 	<compatibility xmlns="urn:schemas-microsoft-com:compatibility.v1">
 		<application>
-			<!-- Required for C++ XAML Islands. More details at https://docs.microsoft.com/en-us/windows/apps/desktop/modernize/host-standard-control-with-xaml-islands-cpp#create-a-desktop-application-project -->
+			<!-- Required for C++ XAML Islands. More details at https://learn.microsoft.com/windows/apps/desktop/modernize/host-standard-control-with-xaml-islands-cpp#create-a-desktop-application-project -->
```

Comment:
Do we still use Islands?

Reply:
This one may actually. @jaimecbernardo to verify

Reply:
Indeed we still use them here.